_layouts/default.html

```diff
@@ -65,7 +65,7 @@
 </head>
 <body class="{{layout.tab}} {{page.tab}}">
     <div class="oh-notification">
-        <p> <strong>Get the most out of Flyway v7 - Sign up for our webinar on the Redgate Hub</strong> <a href="https://www.red-gate.com/hub/events/online-events/whats-new-in-flyway-7-0" class="oh-button">▶ Watch now</a></p>  
+        <p> <strong>Share your experience with Database DeVops by taking part in the 2021 State of Database DevOps survey</strong> <a href="https://www.surveymonkey.com/r/FJ65NDD" class="oh-button">Take the survey</a></p>  
```

Comment:
You've got a typo in there - `DeVops` should be `DevOps` :)

Reply:
It's fine. It's fine.